docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -249,13 +250,15 @@ metadata:
   name: nova-compute-extraconfig
   namespace: openstack
 spec:
-  label: nova.compute.extraconfig
-  configMaps:
-    - nova-compute-extraconfig
-    - ceph-nova
-  secrets:
-    - nova-cell1-compute-config
-    - nova-migration-ssh-key
+  dataSources:
+    - secretRef:
+        name: nova-cell1-compute-config
+    - secretRef:
+        name: nova-migration-ssh-key
+    - configMapRef:
+        name: nova-compute-extraconfig
+    - configMapRef:
+        name: ceph-nova
```

Comment:
should we add it as optional in the default nova service?

Reply:
no infact i would like to do the opiciete and adjust all our test code to use nova-extra-config

that is the name that we want to standardise on in our docs and test code.